Doc/library/compileall.rst

```diff
@@ -185,10 +185,13 @@ Public functions
    .. versionchanged:: 3.7
       The *invalidation_mode* parameter was added.
 
+   .. versionchanged:: 3.7.2
+      The *invalidation_mode* parameter's default value is updated to None.
+
    .. versionchanged:: 3.8
       Setting *workers* to 0 now chooses the optimal number of cores.
 
-.. function:: compile_file(fullname, ddir=None, force=False, rx=None, quiet=0, legacy=False, optimize=-1, invalidation_mode=py_compile.PycInvalidationMode.TIMESTAMP)
+.. function:: compile_file(fullname, ddir=None, force=False, rx=None, quiet=0, legacy=False, optimize=-1, invalidation_mode=None)
```

Comment:
It seems the `versionchanged` tag was to be added for this method as well.

Reply:
ops, i forgot this line. Thanks, Sanyam :)